int32 exposure
---
bool result
